<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CheckButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>检查</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InputGraphButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入原图</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="LogBrowser">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>601</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearLogButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="InputGraph">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="InputSubGraph">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="InputSubGraphButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入子图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewCheckerButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>生成检查器</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>结点数据类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="IntFormatButton">
       <property name="text">
        <string>整形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="StringFormatButton">
       <property name="text">
        <string>字符串</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
